src/features/offer/components/OfferBody/OfferBody.tsx

```diff
@@ -168,6 +190,16 @@ export const OfferBody: FunctionComponent<Props> = ({
     analytics.logConsultItinerary({ offerId, from: 'offer' })
   }, [offerId])
 
+  const onEndReached = useCallback(() => {
+    if (data && hasNextPage) {
+      void (async () => {
+        await fetchNextPage()
+      })()
+    }
+  }, [data, fetchNextPage, hasNextPage])
```

Comment:
Un peu bizarre ça non ?

```suggestion 
  const onEndReached = useCallback(async () => {
    if (data && hasNextPage) {
      await fetchNextPage()
    }
  }, [data, fetchNextPage, hasNextPage])
```

ou alors si c'est le fait que la fonction soit asynchrone qui pose problème tu peux peut-être faire
```suggestion 
  const onEndReached = useCallback() => {
    if (data && hasNextPage) {
      void fetchNextPage()
    }
  }, [data, fetchNextPage, hasNextPage])
```

Reply:
Ca ne gène personne au Pass ce délire de Sonar ?
Je veux dire, c'est archi relou, il faut laisser JavaScript faire nan ?

Reply:
Tu pourras l'évoquer en synchro :)